src/pybind/mgr/dashboard_v2/frontend/src/app/core/auth/logout/logout.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { HttpClient } from '@angular/common/http';
+import { AuthService } from '../../../shared/services/auth.service';
+
+@Component({
+  selector: 'oa-logout',
+  templateUrl: './logout.component.html',
+  styleUrls: ['./logout.component.scss']
+})
+export class LogoutComponent implements OnInit {
+
+  constructor(private authService: AuthService,
+              private router: Router,
+              private http: HttpClient) { }
+
+  ngOnInit() {
+  }
+
+  logout() {
+    this.http.delete('/api/auth').subscribe(() => {
```

Comment:
Same comment as in the `login.component.ts`, the HTTP request should be done from the `AuthService` code,

Reply:
Done